src/State/AuthFlow/actions.js

```diff
@@ -0,0 +1,23 @@
+import * as types from './types';
+
+// assuming that the token is encoded?
```

Comment:
I feel like we could nuke the comment, or just specify that we know it is encoded to help with future clarity. Sorry this is a super tiny nitpick 😢

Reply:
yeah I left that in as I was thinking through the process, will nuke comment w/ no mercy